config/controller.yaml

```diff
@@ -74,6 +74,9 @@ spec:
           # The shell image must be root in order to create directories and copy files to PVCs.
           # As of May 19, 2020
           "-shell-image", "gcr.io/distroless/base:debug@sha256:f79e093f9ba639c957ee857b1ad57ae5046c328998bf8f72b30081db4d8edbe4",
+
+          # This image is used as a placeholder pod, the Affinity Assistant
+          "-affinity-assistant-image", "nginx",
```

Comment:
Should we "pin" a nginx image with digest here ? (as the above images)?

Reply:
I was thinking about it, but we don't depend on any functionality - just that it is a "service" (that does not terminate). But I don't know if it is better to pin?

Reply:
If it just needs to be an image that runs indefinitely, can it be `busybox` with `command: ['sleep', 'infinity']`?

Reply:
yes, but we want to externalize the image here, so that e.g. openshift-pipelines can use a custom image and registry. I think a custom Go-binary and built from "scratch" would be best, as in #2640

Reply:
yeah, this PR is *just* about making sure we don't hardcode the image (in the release to come) :angel:

Reply:
sgtm, flagging is better than hard-coding, then we can work on making it a simpler Tekton-owned image.

Reply:
> But I don't know if it is better to pin?

i think it makes sense to pin - without pinning images we depend on we sometimes end up in states where things are breaking, and often for reasons we don't really care about (e.g. https://github.com/tektoncd/pipeline/issues/2738) - pinning reduces the number of things that can change on us and cause us problems :D